utilities/cwp/custom-war-packager.inc

```diff
@@ -0,0 +1,137 @@
+# function to download the CWP jar
+# $1: Working directory
+# $2: CWP version to use
+download_cwp() {
+
+	if [ "$#" -gt 1 ]
+	then
+		cwp_jar_file="$1/.cwp-build/cwp-cli-$2.jar"
+
+		rm -rf "$1/.cwp-build"
+		mkdir -p "$1/.cwp-build"
+
+		if [[ $2 == *"SNAPSHOT"* ]]
+		then
+			wget -O "$cwp_jar_file" https://repo.jenkins-ci.org/snapshots/io/jenkins/tools/custom-war-packager/custom-war-packager-cli/$2/custom-war-packager-cli-$2-jar-with-dependencies.jar
+		else
+			if [[ $2 == *"rc"* ]]
```

Comment:
So we should avoid manual releases with versions like `2.0-rc1`. Should be good enough for now, but a TODO comment would not hurt

Reply:
I don't follow you, Oleg. Maybe I'm missing something with the incremental releases, but I thought the RC version were incremental.